ArcGISRuntimeSDKQt_CppSamples/AR/DisplayScenesInTabletopAR/DisplayScenesInTabletopAR.pro

```diff
@@ -90,3 +91,4 @@ contains(ANDROID_TARGET_ARCH,arm64-v8a) {
     ANDROID_PACKAGE_SOURCE_DIR = \
         $$PWD/android
 }
+}
```

Comment:
This is a suspicious new brace. Please confirm this is the right place for this. If it is, the indenting needs to be fixed.